contrib/experimental/great_expectations_experimental/expectations/expect_column_discrete_entropy_to_be_between.py

```diff
@@ -309,9 +309,8 @@ def _prescriptive_renderer(
         **kwargs,
     ):
         runtime_configuration = runtime_configuration or {}
-        include_column_name = runtime_configuration.get("include_column_name", True)
         include_column_name = (
-            include_column_name if include_column_name is not None else True
```

Comment:
Thoughts on this suggestion?
```python
include_column_name = runtime_configuration.get("include_column_name", True)
if include_column_name is None:
    include_column_name = True
```

I'm always wary of one liners like this but I think this is readable enough - will defer to you.
Also, would it be valuable to encapsulate this logic in a util method such that we can share the logic across the hierarchy?

Reply:
Maybe a util method is the way to go. I'm just looking for a way for users to understand what this is doing. It's quite odd.

Reply:
Per our offline convo, we will encapsulate this and other duplicative logic in these renderer templates in a class method.